api/src/test/java/org/openmrs/module/sync/SyncORUR01HandlerTest.java

```diff
@@ -1,92 +0,0 @@
-/**
```

Comment:
I don't remember why I deleted this test.  I can look into this again, but do we really need this anymore?